recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+do_install_append () {
+    rm ${D}${sysconfdir}/${PN}/config.conf
+    install -d ${D}/mnt/config/hawkbit
+    install -m 644 ${S}/config.conf.example ${D}/mnt/config/hawkbit/config.cfg
+    ln -s /mnt/config/hawkbit/config.cfg ${D}/${sysconfdir}/rauc-hawkbit-updater/config.conf
+}
+
+FILES_${PN} = " \
+    ${bindir}/rauc-hawkbit-updater \
+    /mnt/config/hawkbit/config.cfg \
+    ${sysconfdir}/rauc-hawkbit-updater/config.conf \
+"
```

Comment:
Do not overwrite `FILES` in a bbappend. Rather append with `+=` only the files you added with this bbappend. In this case this is only the symbolic link.

Reply:
apparently `FILES` isn't needed at all when creating a link.